kanidmd/src/lib/actors/v1_write.rs

```diff
@@ -1011,6 +1009,61 @@ impl QueryServerWriteV1 {
         res
     }
 
+    #[instrument(
+        level = "trace",
+        name = "idmaccountpersonset",
+        skip(self, uat, uuid_or_name, eventid)
+        fields(uuid = ?eventid)
+    )]
+    pub async fn handle_idmaccountpersonset(
```

Comment:
The duplication between this and `idmaccountpersonextend` doesn't feel great, but it also seems sorta reasonable.

The idea of an admin initializing an account to a "person" with some initial values makes a lot of sense, and it also makes sense there'd be some overlap with "initialize a person" and "update a person".

If you have an idea for how to better structure this code to make it feel less duplicated, I'm happy to hear ideas!

Reply:
I don't today. But I think once we start to work on scim it might become clear how to make it better :)